### PR TITLE
Spider now contains real user picture

### PR DESCRIPTION
**Pull request by borek2**
Spider still had my head on it always. Changed it to the actual user (Try it out by adding a different user to the url parameter `?userName=buster` for example)

### CODE REVIEW

File: src/modules/all-prs-force-layout.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ define(function () {
         yAxis: false,
         yRightAxis: false,
         data: [{
-            'serviceCall': function () { return new ForceLayoutAggregator(globalUserName); },
+            'serviceCall': function () { return new ForceLayoutAggregator(globalUserName, "GitHub"); },
```
Comment (agudek) [inline src/modules/all-prs-force-layout.js]: Is it not possible to change this into a global platform variable?

Comment (borek2) [inline src/modules/all-prs-force-layout.js]: Anything is possible!

magic

Comment (borek2) [inline src/modules/all-prs-force-layout.js]: will do 💃

**Comment by agudek** [comment]
Good to see it implemented this quickly. Will see if it works in an hour or two.

**Pull request merged**